test_scripts/UpdateVideoStreamingCapabilities/004_OnAppCapabilityUpdated_capability_parameter_has_additionalVideoStreamingCapabilities_only.lua

```diff
@@ -0,0 +1,58 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0296-Update-video-streaming-capabilities-during-ignition-cycle.md
+--
+-- Description:
+-- Processing OnAppCapabilityUpdated notification from mobile to HMI
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. App1 with NAVIGATION appHMIType is registered
+-- 3. OnAppCapabilityUpdated notification is alowed by policy for App1
+--
+-- Sequence:
+-- 1. App1 sends OnAppCapabilityUpdated for VIDEO_STREAMING capability type with videoStreamingCapability 
+--    parameter that contains additionalVideoStreamingCapabilities parameter only
+-- 2. SDL sends OnAppCapabilityUpdated notification to the HMI
+
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local common = require('test_scripts/UpdateVideoStreamingCapabilities/common')
+
+local appCapability = {
+  appCapability = {
+  	appCapabilityType = "VIDEO_STREAMING",
+    videoStreamingCapability = {
+      additionalVideoStreamingCapabilities = {
```

Comment:
please use the value from common file like 
`additionalVideoStreamingCapabilities = { common.videoStreamingCapabilityWithOutAddVSC }`